src/input/source/hidraw/zotac_zone.rs

```diff
@@ -0,0 +1,42 @@
+use std::{error::Error, fmt::Debug};
+
+use crate::{
+    drivers::zotac_zone::driver::Driver,
+    input::{
+        capability::Capability,
+        event::native::NativeEvent,
+        source::{InputError, SourceInputDevice, SourceOutputDevice},
+    },
+    udev::device::UdevDevice,
+};
+
+/// XpadUhid source device implementation
+pub struct ZotacZone {
+    _driver: Driver,
+}
+
+impl ZotacZone {
+    /// Create a new source device with the given udev
+    /// device information
+    pub fn new(device_info: UdevDevice) -> Result<Self, Box<dyn Error + Send + Sync>> {
+        let driver = Driver::new(device_info)?;
+        Ok(Self { _driver: driver })
+    }
+}
+
+impl Debug for ZotacZone {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("ZotacZone").finish()
+    }
+}
+impl SourceInputDevice for ZotacZone {
+    fn poll(&mut self) -> Result<Vec<NativeEvent>, InputError> {
+        Ok(vec![])
+    }
+
+    fn get_capabilities(&self) -> Result<Vec<Capability>, InputError> {
+        Ok(vec![])
```

Comment:
I assume that this is done because there are no capabilities/polls for the config interface, and everything else is done via evdev?

Reply:
Yeah. Very same as Ally